.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn lint-staged
```

Comment:
Сейчас при коммите говорит 
```
hint: The '.husky/pre-commit' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`
```
`.husky/pre-commit` остался неисполняемым. Он вообще ещё нужен?

Reply:
Нужен
При коммите запускает lint-staged